CMakeLists.txt

```diff
@@ -614,7 +614,7 @@ if (BUILD_PYTHON)
     endif ()
   endif ()
 
-  find_package (PythonInterp)
+  find_package (PythonInterp 3.5)
```

Comment:
Does this mean that we will need precisely Python 3.5 to run OpenTURNS?

Reply:
that's >= 3.5